evaltools/scoring/partisan.py

```diff
@@ -37,51 +37,55 @@ def _party_wins_by_district(part: Partition, election_cols: Iterable[str], party
     districts = [d for d in sorted(part.parts.keys()) if d != -1]
     return {d: int(d_wins) for d, d_wins in zip(districts, stability)}
 
+def _aggregate_seats(part: Partition, election_cols: Iterable[str], party: str) -> PlanWideScoreValue:
+    stability = _election_stability(part, tuple(election_cols), party)
+    return int(stability.sum())
+
 def _seats(part: Partition, election_cols: Iterable[str], party: str, mean: bool = False) -> ScoreValue:
-    result = {part[e].election.name: sum([part[e].won(party, d) for d in part.parts.keys() if d != -1]) for e in election_cols}
-    return np.mean(list(result.values())) if mean else result
+    result = {e: sum([part[e].won(party, d) for d in part.parts.keys() if d != -1]) for e in election_cols}
+    return int(np.mean(list(result.values()))) if mean else result
```

Comment:
I see why we'd want to `int()` so that it makes sense as a count of seats, but if someone wants the mean don't we think they'd be okay with it being a `float`? I could go either way...

Reply:
`numpy.mean` has a return type of `numpy.float64` which can cause issues with serializing to json.  This forces it to be a default python float type.